File: src/main/scala/jetbrains/zeppelin/toolwindow/TextLogger.scala
package jetbrains.zeppelin.toolwindow

import jetbrains.zeppelin.utils.ZeppelinOutput

/**
  * Implementation of the output for the tests
  */
class TextLogger extends ZeppelinOutput {
  override def printMessage(msg: String): Unit = println(msg)

  override def printError(msg: String): Unit = println(msg)
}

File: zeppelin/src/org/intellij/plugin/zeppelin/idea/toolwindow/TextLogger.kt
package org.intellij.plugin.zeppelin.idea.toolwindow

import org.intellij.plugin.zeppelin.utils.ZeppelinOutput

/**
 * Implementation of the output for the tests
 */
class TextLogger : ZeppelinOutput {
    override fun printError(msg: String): Unit = println(msg)
    override fun printMessage(msg: String): Unit = println(msg)
}